### ISSUE TITLE
Uncaught SyntaxError when integrating a build from the online builder when using `npm add file:./ckeditor5` in Vue

### ISSUE THREAD

**Issue by mabryl** [ckeditor/ckeditor5-vue#263]
## 📝 Provide detailed reproduction steps (if any)

1. Try to integrate a build from the online builder in accordance with [our docs](https://ckeditor.com/docs/ckeditor5/latest/installation/integrations/vuejs-v3.html#integrating-a-build-from-the-online-builder)
2. In the step that describes adding the package located in the `ckeditor5` directory as a dependency, use npm instead of yarn like so: `npm add file:./ckeditor5`
3. Run the app

### ✔️ Expected result

You should be able to see the editor running in Vue.

### ❌ Actual result

You get a blank page and an error in the console that reads: `Uncaught SyntaxError: The requested module '/ckeditor5/build/ckeditor.js' does not provide an export named 'default' (at App.vue:8:10)`

### 🤔 Additional details

As stated in the reproduction steps, this only happens when using npm. Using yarn via `yarn add file:./ckeditor5` does not produce this issue.

Marking this as a regression because I'm fairly sure that `npm add file:./ckeditor5` worked in an older version of the editor.

**Comment by Witoso** [comment]
@mabryl could you specify your npm and node version?

**Comment by mabryl** [comment]
Node v18.17.1
npm 9.6.7

Should I update and try again @Witoso?

**Comment by Witoso** [comment]
I don't think it would help. What's your bundler in the app? If it's vite, there may be some additional configuration needed. I think this is a bundler issue, and this worked in Angular/React docs, but doesn't in Vue due to different bundlers.

There's a resolved issue for this on [SO](https://stackoverflow.com/questions/74559310/uncaught-syntaxerror-the-requested-module-ckeditor5-build-ckeditor-js-does-n).

**Comment by mabryl** [comment]
Yup, the bundler is the default one when using `npm create vue@latest`, so it's Vite.

I'll see if the issue persists when using webpack, for example.

**Comment by DedicatedManagers** [comment]
@Witoso I was able to use [your link](https://github.com/ckeditor/ckeditor5-vue/issues/263#issuecomment-1782991189) to get the editor to show up in the dev environment, but when I go to build `npx vite build` it errors out and doesn't compile.  Nobody seems to have discussed that part of the issue.

Thanks.

**Comment by Witoso** [comment]
@DedicatedManagers could you also check [the comment](https://github.com/ckeditor/ckeditor5/issues/9807#issuecomment-1755980661) that was shared in our community with a config?

**Comment by FrankyXie** [comment]
> @Witoso I was able to use [your link](https://github.com/ckeditor/ckeditor5-vue/issues/263#issuecomment-1782991189) to get the editor to show up in the dev environment, but deploy, I get the error:
> 
> index-f0797983.js:1 TypeError: this.editor.create is not a function

did you find any solution?
